src/rusage.rs

```diff
@@ -9,10 +9,10 @@ pub(crate) struct Rusage {
     pub(crate) max_res_size: f64,
 }
 
-fn ms_from_timeval(tv: timeval) -> f64 {
+fn μs_from_timeval(tv: timeval) -> f64 {
     let seconds = tv.tv_sec;
-    let ms = tv.tv_usec as i64;
-    let val = seconds * 1000000 + ms;
+    let μs = tv.tv_usec as i64;
```

Comment:
I changed the variable name as `ms` means milliseconds but these are microseconds.